no-computer.md

```diff
@@ -1,69 +1,60 @@
-# No computer project
+# No computer project: Badminton doubles hand-signals
 
 
 ## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
+This section describes what the no-computer DSL is and who uses it
 
+The fastest sport in the world. The only time badminton doubles players drop from that intensity is to serve, and it's one of the most important aspects of doubles. It dictates which side will have the advantage from the start, and the key part to serving well is communication between partners. But they can't just talk to each other, othewise the opponents would know exactly what they are about to do. Enter the DSL: hand signs behind the back. 
```

Comment:
Is there a reason they need to use a physical code rather than a verbal code. I understand they can't just talk to each other in plain language, but could they not agree on a code before hand (e.g. server says a number, 1 means frontcourt, 2 means lob serve, 3 means ...).  This seems at least as easy as memorizing hand symbols.

Or is this kind of communication discouraged / against the rules.  If so, would it also be against the rules (or spirit of the rules) to use these hand motion signals?